### ISSUE TITLE
$page->goto returns NULL for some URLs

### ISSUE THREAD

**Issue by 7starsone** [extractr-io/puphpeteer#2]
Hello,
it works fine with the example.com but if eg. you try the domain icecat.de as target,
the page content returns **NULL**

I'm on CentOS 7 + PHP 7.1 and Nodejs 9
These are the script lines:
```
require 'vendor/autoload.php';
use ExtractrIo\Puphpeteer\Puppeteer; 

$puppeteer = new Puppeteer;
$browser = $puppeteer->launch([
    'args' => ['--no-sandbox', '--disable-setuid-sandbox'],
]);
$page = $browser->newPage();
$response = $page->goto('https://icecat.de');
var_dump($page->content());
$browser->close();
```
there are other not working tested URLs but I don't write them here and I kindly ask you to find the cause and fix it to work with any site, please
Thank you

**Comment by 7starsone** [comment]
Hello, any news about it, please? Thanks

**Comment by nesk** [comment]
Hello, currently I don't have much time to work on it. The issue is pretty complicated and it seems to happen because the data payload transfered between the node process and PHP is too big. Actually, this is an issue with [extractr-io/rialto](https://github.com/extractr-io/rialto) and not PuPHPeteer, I'm closing this issue and creating a new one.

**Comment by nesk** [comment]
Until its fixed, if you really want to retrieve the HTML content, you should instead make a simple HTTP request with [Guzzle](http://docs.guzzlephp.org/en/stable/) (or [cURL](http://php.net/manual/fr/book.curl.php), etc…).

PuPHPeteer should work for other instructions with smaller payloads.

**Comment by nesk** [comment]
@7starsone FYI, I've fixed the bug in v0.2.1 🙂
